<launch>

  <!-- Arguments -->
  <arg name="rosbag_filename" default="/home/bjoshi/cave_gt.bag"/>
  <param name="/use_sim_time" value="true"/>

  <node name="add_sonar" pkg="utils_ros" type="add_sonar" output="screen" > 
    <param name="rosbag_filename" value="$(arg rosbag_filename)"/>
  </node>

</launch>
